modules/client/test/integration/tokenVoting-client/methods.test.ts

```diff
@@ -1442,6 +1446,54 @@ describe("Token Voting Client", () => {
         const token = await client.methods.getToken(pluginAddress);
         expect(token).toBe(null);
       });
+
+      it("Should check if a ERC20 is compatible with the TokenVotingSetup and return needs wrapping", async () => {
+        const ctx = new Context(contextParamsLocalChain);
+        const client = new TokenVotingClient(ctx);
+        const erc20Token = await deployErc20();
+        const compatibility = await client.methods.isTokenVotingCompatibleToken(
+          erc20Token.address,
+        );
+        expect(compatibility).toBe(
+          TokenVotingTokenCompatibility.NEEDS_WRAPPING,
+        );
+      });
+      it("Should check if ERC721 is compatible with governance and return incompatible", async () => {
+        const ctx = new Context(contextParamsLocalChain);
+        const client = new TokenVotingClient(ctx);
+        const erc721Token = await deployErc721();
+        const compatibility = await client.methods.isTokenVotingCompatibleToken(
+          erc721Token.address,
+        );
+        expect(compatibility).toBe(TokenVotingTokenCompatibility.INCOMPATIBLE);
+      });
+      it("Should be called with an invalid address and throw InvalidAddressError", async () => {
+        const ctx = new Context(contextParamsLocalChain);
+        const client = new TokenVotingClient(ctx);
+        expect(() =>
```

Comment:
missing `await`

Reply:
Let's add a linting CI workflow to the SDK and OSx repo as we did on the plugin template repo @Rekard0